demo/index.js

```diff
@@ -1,10 +1,15 @@
 import window from 'global/window';
 import document from 'global/document';
 
-// the brightcove player of your choosing, defined in webpack.config.js
-// will be prepended to your bundle. The bc function will be available in your
-// entry point.
-const player = window.bc(document.getElementById('vid1'));
+// The Brightcove Player of your choosing, defined in webpack.config.js,
+// will be prepended to your bundle. The bc function will be available in
+// your entry point.
+const player = window.bc(document.querySelector('video'));
 
-/* eslint-disable no-console */
-console.log(player);
+// Expose the player globally, so we can easily interact with it in the
+// browser console. It is not recommended that you do this in production!
+window.player = player;
+
+// Set a poster image and an MP4 source, for demo purposes.
+player.poster('http://vjs.zencdn.net/v/oceans.png');
```

Comment:
If its a player with a catalog source this could make it seem like it isn't working.

Reply:
I don't think we reliably can detect that - especially with all the different player versions out there. I think we should let it be at, if someone has a complaint, we can consider addressing it at that point.